Project-Hotpot/Project-Hotpot/PFSong.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  PFSong.swift
+//  Project-Hotpot
+//
+//  Created by Harini Sundaram on 7/14/22.
+//
+
+import UIKit
+import Parse
+
+//will store Songs as PFSong objects
+class PFSong: PFObject, PFSubclassing {
+    @NSManaged var name : String
+    @NSManaged var URI: String
```

Comment:
You can check naming standards but generally we only use caps for constants, consider changing to 'uri'